### PR TITLE
Adding Danilo

### PR DESCRIPTION
**Pull request by nilodna**

### CODE REVIEW

File: places.geojson [hunk 1]
```diff
@@ -46,6 +46,15 @@
...
                 "description": "An ancient volcanic plug formed by magma, this 581-foot rock is known as a harbor entrance landmark (also a place which Chris visited often when he was young)."
             }
-        }        
+        },
+        {
+          "type": "Feature",
+          "geometry": { "type": "Point", "coordinates": [ -42.0391845703125,-22.98873816096073] },
+          "properties": {
+            "name": "Cabo Frio Location",
+            "type: "Upwelling zone",
```
Comment (abarciauskas-bgse) [inline places.geojson]: Thanks @nilodna ! I think this needs a quotation mark
```suggestion
            "type": "Upwelling zone",

```

Comment (nilodna) [reply places.geojson]: Thanks @abarciauskas-bgse. I think I solved it now.

File: places.geojson [hunk 2]
```diff
@@ -46,6 +46,15 @@
...
+        },
+        {
+          "type": "Feature",
+          "geometry": { "type": "Point", "coordinates": [ -42.0391845703125,-22.98873816096073] },
+          "properties": {
+            "name": "Cabo Frio Location",
+            "type": "Upwelling zone",
+            "location": "Cabo Frio, Brazil",
+            "description": "it is hard to swimm in there."
+        }
```
Comment (abarciauskas-bgse) [inline places.geojson]: sorry I missed this earlier, but I think we need another curly brace here
```suggestion
        }
    }
```

File: places.geojson [hunk 3]
```diff
@@ -46,6 +46,15 @@
...
-        }        
+        },
+        {
+          "type": "Feature",
+          "geometry": { "type": "Point", "coordinates": [ -42.0391845703125,-22.98873816096073] },
+          "properties": {
+            "name": "Cabo Frio Location",
+            "type": "Upwelling zone",
+            "location": "Cabo Frio, Brazil",
+            "description": "it is hard to swimm in there."
```
Comment (abarciauskas-bgse) [inline places.geojson]:
```suggestion
            "description": "it is hard to swim in there."

```

**Pull request merged**